src/routes/delegates/delegates.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Controller, Get, Param, Query } from '@nestjs/common';
+import { DelegatesService } from './delegates.service';
+import { ApiImplicitQuery } from '@nestjs/swagger/dist/decorators/api-implicit-query.decorator';
+import { Delegate } from './entities/delegate.entity';
+import { DelegatePage } from './entities/delegate.page.entity';
+import { RouteUrlDecorator } from '../common/decorators/route.url.decorator';
+import { PaginationDataDecorator } from '../common/decorators/pagination.data.decorator';
+import { PaginationData } from '../common/pagination/pagination.data';
+import { Page } from '../common/entities/page.entity';
+
+@ApiTags('Delegates')
+@Controller({
+  version: '1',
+})
+export class DelegatesController {
+  constructor(private readonly service: DelegatesService) {}
+
+  @ApiOkResponse({ type: DelegatePage })
+  @ApiImplicitQuery({
+    name: 'safe',
+    required: false,
+    type: String,
+  })
+  @ApiImplicitQuery({
+    name: 'delegate',
+    required: false,
+    type: String,
+  })
+  @ApiImplicitQuery({
+    name: 'delegator',
+    required: false,
+    type: String,
+  })
+  @ApiImplicitQuery({
+    name: 'label',
+    required: false,
+    type: String,
+  })
+  @ApiImplicitQuery({
+    name: 'cursor',
+    required: false,
+    type: String,
+  })
+  @Get('chains/:chainId/delegates')
+  async getDelegates(
```

Comment:
We should validate here that at least one query parameter between (safe, delegate, delegator or label) is not `null` because if not, the `tx-service` returns `400 bad request `

Reply:
If we agree on using TS guards as validation tool for request, you can find [here](https://github.com/5afe/safe-client-gateway-nest/pull/132/commits/206ab186563e3f3945facd881c6da292aca9231e#diff-12bd9e35019ab1c8ce38b567178f33911bc1253984905fb8b274f6779a804d8a) an example (pending to be reviewed). 

In my opinion would be nice to map the values in a DTO and validate it, but you could also implement the validation at `DelegatesService` level manually, no strong opinion here. And probably we could do it in a separate issue/PR as well if you prefer.